providers/burst-sms/src/lib/burstSms.provider.spec.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
```

Comment:
I missed to ask for a rename of the test file to: `burst-sms.provider.spec.ts`. Apologies for that.